widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/HasOngoingSecureConversationUseCase.kt

```diff
@@ -22,12 +20,19 @@ internal class HasOngoingSecureConversationUseCase(
             secureConversationsRepository.unreadMessagesCountObservable,
             engagementStateUseCase()
         ) { pendingSecureConversations, unreadMessagesCount, state ->
-            isAuthenticatedUseCase() &&
-                (pendingSecureConversations || unreadMessagesCount > 0 || state is State.TransferredToSecureConversation)
+            pendingSecureConversations || unreadMessagesCount > 0 || state is State.TransferredToSecureConversation
```

Comment:
Removed the `isAuthenticatedUseCase()` check, as it was included in the unread_messages_count observable on the core SDK side.

Reply:
I don't see a problem if under the hood as I understood it is still taken into account.

Reply:
In my opinion we can keep `isAuthenticatedUseCase()` here because it's directly related to HasOngoing**Secure**ConversationUseCase. Unlike the `unread_messages_count` observable - it's name doesn't say that it's related to Secure Conversations.

Reply:
So, in general, we don't need to check for authentication here.